src/backend/distributed/utils/citus_ruleutils.c

```diff
@@ -473,8 +473,7 @@ EnsureRelationKindSupported(Oid relationId)
 							relationKind == RELKIND_FOREIGN_TABLE;
 
 	/*
-	 * Citus doesn't support bare inherited tables (i.e., not a partition or
-	 * partitioned table)
+	 * Citus doesn't support inherited tables
```

Comment:
I don't think this new sentence is accurate. Partitioned tables are also classified as inherited tables, and there is also another concept of [inherited tables](https://www.postgresql.org/docs/11/tutorial-inheritance.html) which Citus doesn't support. And, this check is for that.

Also, this doesn't seem to be related to the PR, so can you remove it?